src/plugins/profiling/server/routes/search_flamechart.ts

```diff
@@ -157,18 +167,23 @@ async function queryFlameGraph(
           // It reduces the query time by avoiding unneeded searches.
           querystring: {
             pre_filter_shard_size: 1,
+            filter_path:
+              'aggregations.group_by.buckets.key,aggregations.group_by.buckets.count,aggregations.total_count,_shards.failures',
```

Comment:
What's this?

Reply:
This is `filter_path` - removing everything else from the JSON response (there is a lot of unneeded cruft). This reduces marshalling/unmarshalling overhead.